sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -347,12 +352,15 @@ private void setArchitectures(final @NotNull Device device) {
     if (memInfo != null) {
       // in bytes
       device.setMemorySize(getMemorySize(memInfo));
-      device.setFreeMemory(memInfo.availMem);
-      device.setLowMemory(memInfo.lowMemory);
+      if (applyScopeData) {
+        device.setFreeMemory(memInfo.availMem);
+        device.setLowMemory(memInfo.lowMemory);
+      }
       // there are runtime.totalMemory() and runtime.freeMemory(), but I kept the same for
       // compatibility
     }
 
+    // likely to be applied only if applyScopeData
```

Comment:
the chances that the user opens the app again right away is high, so this won't change much